### ISSUE TITLE
Error when trying to upload playlist

### ISSUE THREAD

**Issue by mtsbh** [Marekkon5/onetagger#148]
Hi, guys, 
I've got this error when I try to upload .m3u playlist into AutoTag. 

![image](https://user-images.githubusercontent.com/108582199/194181522-58f70869-2670-4078-8e33-34fff1817010.png)

There's the playlist. - https://easyupload.io/zutz14 (I'm not allowed to attach m3u file here)

Also, when I try with m3u8- it doesn't even drop it in 1T. It stays blank and nothing more.

**Comment by Marekkon5** [comment]
Hello, the file you've sent contains invalid characters (so 1T can't read it). The issues are on lines: 478-479, 488-491, 588-589 (if you remove those, the playlist works properly).  Which app created this file? Maybe you can set UTF-8 encoding there.

Also I've fixed playlist drag and drops in the latest commit. You can get the binary from the Actions tab to try it out. M3U8 should work as well.

**Comment by mtsbh** [comment]
Hi! I tried with both -  foobar and aimp. The result was the same. 

Thank you! Will try tonight

**Comment by Marekkon5** [comment]
Maybe the actual filenames have weird characters in them:

```
D:\#HARD\#\onur \F6zer - crescendo.aif
D:\#HARD\#\onur \F6zer, tc studio - lapona enigel.aiff
D:\#HARD\#\onur o\A8zer & arapu - rec05.aif
D:\#HARD\#\roomorph - num\E9ro.aif
```

**Comment by mtsbh** [comment]
Yes. I see. It's because of the artist's name  - onur özer. The Turkish language has those types of letters - with two dots.

**Comment by Marekkon5** [comment]
You can maybe try renaming them to not have the special letters.
